app/assets/stylesheets/components/_cards.scss

```diff
@@ -1,35 +1,114 @@
-// this has to be moved to container later, put it here because of conflicts
-.arts-container {
-  margin: auto;
-  width: 90%;
-  padding: 10px;
-    a {
-      text-decoration: none;
-      color: white;
-    }
+// does this has to be moved to container later?
+.category-container {
+  margin-top: 20px !important;
+  width: 100%;
+  display: flex;
+  gap: 10px;
+  overflow: hidden;
+  overflow-x: scroll;
+
+  &::-webkit-scrollbar {
+    height: 0px;
+  }
+
+  @media screen and (min-width: $small) {
+
+  }
+
+  @media screen and (min-width: $large) {
+    justify-content: center;
+  }
+
+}
```

Comment:
The card css component should only contain the css for the card itself. Not what's around it.

You need to make a new file in stylesheets/arts/_index.scss and put everything that is related to the index page and that is not a card inside there.

This css selector should be in this new file